zone.go

```diff
@@ -497,6 +502,34 @@ func (api *API) ZoneAnalyticsByColocation(zoneID string, options ZoneAnalyticsOp
 
 // Zone Settings
 // https://api.cloudflare.com/#zone-settings-for-a-zone-get-all-zone-settings
+func (api *API) GetZoneSettings(zoneID string) ([]ZoneSetting, error) {
```

Comment:
Can you run `golint` and add method comments to any public methods please? Take a look at other methods we've recently updated for an example.